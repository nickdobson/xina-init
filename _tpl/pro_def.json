{
	"action" : "create",
	"create" : "group",
	"teams"  : ["@{project_name}"],
	"parent" : "@{project_name}",
	"group"  : {
		"name"  : "def",
		"label" : "Definitions",
		"desc"  : "@{project_desc} definitions group."
	}
}