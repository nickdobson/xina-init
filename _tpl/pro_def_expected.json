{
	"action"   : "create",
	"create"   : "database",
	"teams"    : ["@{project_name}"],
	"parent"   : "@{project_name}.def",
	"database" : {
		"name"      : "expected",
		"label"     : "Expected",
		"desc"      : "@{project_desc} housekeeping expected values database.",
		"format"    : "{name} {models}",
		"singular"  : "value",
		"backup"    : true,
		"event"     : true,
		"link"      : true,
		"log"       : true,
		"notify"    : true,
		"subscribe" : true,
		"tag"       : true,
		"track"     : true,
		"trash"     : true,
		"wall"      : true,
		"fields"    : [
			{
				"name" : "name",
				"label" : "Name",
				"type" : "asciivstring(32)"
			},
			{
				"name" : "models",
				"label" : "Models",
				"type" : "jsonarray",
				"nul"  : true
			},
			{
				"name" : "notes",
				"label" : "Notes",
				"type" : "utf8string",
				"nul"  : true
			},
			{
				"name" : "red_hi",
				"label" : "Red High",
				"type" : "float(8)",
				"nul"  : true
			},
			{
				"name" : "yellow_hi",
				"label" : "Yellow High",
				"type" : "float(8)",
				"nul"  : true
			},
			{
				"name" : "green_hi",
				"label" : "Green High",
				"type" : "float(8)",
				"nul"  : true
			},
			{
				"name" : "green_lo",
				"label" : "Green Low",
				"type" : "float(8)",
				"nul"  : true
			},
			{
				"name" : "yellow_lo",
				"label" : "Yellow Low",
				"type" : "float(8)",
				"nul"  : true
			},
			{
				"name" : "red_lo",
				"label" : "Red Low",
				"type" : "float(8)",
				"nul"  : true
			}
		]
	}
}